.circleci/config.yml

```diff
@@ -50,7 +41,28 @@ jobs:
           paths:
             - ~/.gradle
 
-  test-web:
+  build-image-api:
+    working_directory: ~/marquez
+    machine: true
+    steps:
+      - checkout
+      - run: ./.circleci/get-jdk11.sh
```

Comment:
Move to jdk17 after this one? https://github.com/MarquezProject/marquez/pull/1819

Reply:
Yep, I'm going to hold off merging this PR. Added #1819 as a dependency.